api/elastic.py

```diff
@@ -389,16 +393,32 @@ def createFilterClausesAndAggregations(self, filterParams):
                 'filter', **{'terms': {'editions.formats': formats}}
             )
 
+        if len(govDocFilters) > 0:
+            # onlyGovDoc|noGovDoc|all
+            # if govDocFilter is all: skip
+            # elif govDocFilter is no: is_government_document: false
+            # elif govdocFilter is only: is_gov_doc: true
+            if govFilter is 'noGovDoc':
```

Comment:
`govFilter` is still `None` here, should look at `govDocFilters` which is an array of 2-tuples in the form of `(filterName, filterValue)`. There should only ever be one tuple in the array though, which makes this easier.